src/stylesheets/_input_fields.scss

```diff
@@ -182,9 +184,8 @@
 
     .react-date-picker__inputGroup {
         @extend %input-style;
-
         color: $color-text;
-        font-weight: 400;
+        font-weight: normal;
```

Comment:
I prefer 400 so that we don't end up mixing numbers and string weights.

Reply:
I'll update the rule in stylelint-config accordingly.